FracCalc/src/fracCalc/FracCalc.java

```diff
@@ -16,13 +30,334 @@ public static void main(String[] args)
+    public static String convertToMixedNumber(int numerator, int denominator) {  
+    	//initializes variables
+	int simplifiedNumerator = numerator;
+    	int simplifiedDenominator = denominator;
+
+    	int whole = ((simplifiedNumerator - (simplifiedNumerator % simplifiedDenominator)) / denominator);
+    	int mixedNumerator = (numerator % denominator);
+    	
+    	simplifiedNumerator = mixedNumerator / GCF(mixedNumerator, denominator); // Simplifies numerator
+    	simplifiedDenominator = denominator / GCF(mixedNumerator, denominator); //simplifies denominator
+ 
+    	//if simplified fraction is a whole number, return only the whole number
+    	if ((simplifiedNumerator == 0) & (simplifiedDenominator == 1)) { 
+    		return whole + "";
+    	} else {
+    		//if both are negative, they are actually positive
+    		if (simplifiedNumerator < 0 && simplifiedDenominator < 0) { 
+    			simplifiedNumerator *= -1;
+    			simplifiedDenominator *= -1;
+    		}
+    		
+    		//if one number in the fraction is negative, then the whole fraction is negative
+    		if (simplifiedNumerator < 0 && whole > 0) { 
+    			whole *= -1;
+    			simplifiedNumerator *= -1;
+    		}
+    		if (simplifiedDenominator < 0) {
+    			if (whole > 0) {
+    				whole *= -1;
+    				simplifiedDenominator *= -1;
+    			}
+    			if (whole == 0 && simplifiedDenominator < 0) {//if denominator is negative, but numerator is not, moves "-" to numerator
+    				simplifiedNumerator *= -1;
+    				simplifiedDenominator *= -1;
+    			}
+    		}
+    		
+    		// if the whole number is negative, remove any other negative signs within the fraction
+    		if (whole < 0) { 
+    			if (simplifiedNumerator < 0) {
+    				simplifiedNumerator *= -1;
+    			}
+    			if (simplifiedDenominator < 0) {
+    				simplifiedDenominator *= -1;
+    			}
+    		}
+    		if (whole == 0) {
+    			return simplifiedNumerator + "/" + simplifiedDenominator;
+    		}
+    		return whole + "_" + simplifiedNumerator + "/" + simplifiedDenominator;
+    	}
     }
+    
+    /* finds the greatest common factor of the denominator and numerator */
+   public static int GCF(int numerator, int denominator) {   
+       if (numerator == 0) { 
+          return denominator;
+          
+        } else if (denominator == 0) { 
+          return numerator; 
+          
+        } else if (numerator == denominator) { 
+            return numerator; 
+          
+        } else if (numerator > denominator) {
+        	//these always need to be positive
+            if (numerator < 0) {
+            	numerator *= -1;
+            }
+            if (denominator < 0) {
+            	denominator *= -1;
+            }
+            
+	        while (numerator != denominator) {
+	        	if (numerator > denominator) {
+	        		numerator -= denominator;
+	        	} else {
+	        		denominator -= numerator;
+	        	}
+	        }
+	        return numerator;
+
+        } else if (numerator < denominator) {
```

Comment:
it would have been easier to normalize your numbers.  you are running the same algorithm twice here.  You could have had a check if denominator > numerator, swap them, and then run the same algorithm.